### PR TITLE
Add com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-1-0

### PR DESCRIPTION
**Pull request by chuwy**

### CODE REVIEW

File: schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-1-0 [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            "maximum": 10000
+        },
+        "compRows": {
+            "type": "integer",
+            "minimum": 1000,
+            "maximum": 1000000000
+        },
+        "roleName": {
+            "type": "string",
+            "minLength": 1
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-1-0]: Why this minLength? The other required strings don't have it...

Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-1-0]: Is it a new Igluctl thing to enforce non-empty strings?

Comment (chuwy) [inline schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-1-0]: No, nothing special about it, but I think we need to start adding more constraints whenever possible.

Comment (chuwy) [inline schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-1-0]: But for sure, it'll help with igluctl severity level.

Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-1-0]: Okay cool, fine by me

**Comment by chuwy** [comment]
@alexanderdean this is ready. 

`minLength` == 20 because of [this doc](http://docs.aws.amazon.com/IAM/latest/APIReference/API_Role.html) (although in fact minLength == 30)

**Comment by alexanderdean** [comment]
Looks good, migrated into a release branch, closing

**Pull request closed without merging**